addons/project/views/project_update_views.xml

```diff
@@ -134,4 +175,123 @@
             </p>
         </field>
     </record>
+
+
+    <record id="project_update_report_view_tree" model="ir.ui.view">
+        <field name="name">project.update.report.view.tree</field>
+        <field name="model">project.update</field>
+        <field name="inherit_id" ref="project_update_view_tree"/>
+        <field name="mode">primary</field>
+        <field name="priority">64</field>
+        <field name="arch" type="xml">
+            <xpath expr="//tree" position="attributes">
+                <attribute name="default_order">date desc</attribute>
+            </xpath>
+            <xpath expr="//field[@name='name']" position="before">
+                <field name="project_id"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="project_update_report_view_form" model="ir.ui.view">
+        <field name="name">project.update.report.view.form</field>
+        <field name="model">project.update</field>
+        <field name="inherit_id" ref="project_update_view_form"/>
+        <field name="mode">primary</field>
+        <field name="priority">64</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='project_id']" position="attributes">
+                <attribute name="invisible" remove="1"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="project_update_report_view_kanban" model="ir.ui.view">
+        <field name="name">project.update.report.view.kanban</field>
+        <field name="model">project.update</field>
+        <field name="inherit_id" ref="project_update_view_kanban"/>
+        <field name="mode">primary</field>
+        <field name="priority">64</field>
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('col-sm-4')]" position="after">
```

Comment:
This doesn't sound like a good identifier to target. Give it an id maybe?

Reply:
I have added an id now :)